src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -7,29 +7,84 @@ incomplete: true
+
+While this has some benefits, it has critical implications for network decentralization and security. Block parameters, like fast block times and big block sizes, increase the difficulty of running a full node—leaving a few "supernodes" responsible for securing the chain. In such a scenario, the possibility of validator collusion or a malicious takeover of the chain increases.  
+
+For blockchains to scale without harming decentralization, running a node must be open to everyone—not necessarily parties with specialized hardware. This is why efforts are underway to ensure everyone can [run a full node](/developers/docs/nodes-and-clients/#why-should-i-run-an-ethereum-node) on the Ethereum network. 
+
+### EVM compatibility {#evm-compatibility}
+
+What makes sidechains useful is their ability to execute contracts developed for the [Ethereum Virtual Machine (EVM)](/developers/docs/evm/). Most EVM-compatible sidechains support smart contracts [written in Solidity](/developers/docs/smart-contracts/languages/), allowing for compatibility with Ethereum Mainnet. 
+
+This means if you want to use your [dapp](/developers/docs/dapps/) on a sidechain, it's just a matter of deploying your [smart contract](/developers/docs/smart-contracts/) to this sidechain. It looks, feels, and acts just like Mainnet – you write contracts in Solidity, and interact with the chain via the Web3 API.
+
+Because sidechains are EVM-compatible, they are considered a useful scaling [scaling solution](/developers/docs/scaling/) for Ethereum-native dapps. With your dapp on a sidechain, users can enjoy lower gas fees and faster transactions, especially if Mainnet is congested. 
+
+However, as explained previously, using a sidechain involves significant trade-offs. Each sidechain is responsible for its security and doesn't inherit Ethereum's security properties. This increases the possibility of malicious behavior which can affect your users or put their funds at risk. 
+
+### Asset movement {#asset-movement}
+
+A crucial feature of sidechains is their ability to facilitate seamless transfer of value from and to Ethereum Mainnet. This interoperability with Ethereum is achieved using a blockchain bridge, which enforces a “two-way peg” between two chains. [Bridges](/bridges/) use a network of smart contracts deployed on both networks to control transfer of funds between Ethereum and sidechains. 
+
+While bridges help users move funds between Ethereum and the sidechain, the assets are not physically moved across the two chains. Instead, a "lock-mint-burn" mechanism is used for transferring value across chains. Here is a description of the actual process:
```

Comment:
Ill do some thinking on this section, but there are a few different types of bridges:

1 - Lock and mint
2 - Burn and mint
3 - Atomic swaps (liquidity providing)

Currently there is a PR up to add some bridging content to the developer docs (https://github.com/ethereum/ethereum-org-website/pull/6291, preview: https://ethereumorgwebsitedev01-developerbridgespage.gtsb.io/en/developers/docs/bridges/)
I don't think we need to go through the actual bridging process on this page and can instead link out to here when it goes live.

Reply:
That PR has been merged in now. 

Thoughts..
1. I think I agree that we should remove that information here, and just leave it in one place on the bridges docs page (now merged in). We can trim most of this out, just mention it uses bridges as a means of migrating assets to and from sidechains, using a variety of mechanisms that you can learn more about in the [bridges documentation]

2. I mentioned this on the other PR, but is there an "burn and unlock" mechanism as alluded to in the below content? Or is the only mechnism used involved burning and "minting" only? If a "burn and unlock" mechanism is used, I would just suggest we add that to the bridge docs in a separate PR.


```suggestion
While bridges help users move funds between Ethereum and the sidechain, the assets are not physically moved across the two chains. Instead, mechanisms that typically involve minting and burning are used for transferring value across chains. More on [how bridges work](/developers/docs/bridges/#how-do-bridges-work).
```